src/types.ts

```diff
@@ -3847,43 +3941,3 @@ const preprocessType = ZodEffects.createWithPreprocess;
 const ostring = () => stringType().optional();
 const onumber = () => numberType().optional();
 const oboolean = () => booleanType().optional();
-
-export {
```

Comment:
Is there a reason for moving all of these exports a lot earlier in the module?

Reply:
No, my bad I think it's an auto save action of my VScode that I didn't noticed.